config/files/root/home/system/edgebox.sh

```diff
@@ -143,16 +158,17 @@ while [ $# -gt 0 ] ; do
         echo "----> Starting Revere Proxy and Service Containers"
         echo ""
         ./ws -s
-	echo ""
-	echo "----> Starting Edgeboxctl"
-	sudo systemctl enable edgeboxctl
-	sudo systemctl start edgeboxctl
-        echo ""
-        echo "---------------------------"
-        echo "| Edgebox Setup Finished  |"
-        echo "---------------------------"
-        echo ""
-        ;;
+        install_edgeboxctl
+        echo ""
+        echo "----> Starting Edgeboxctl"
+        sudo systemctl enable edgeboxctl
+        sudo systemctl start edgeboxctl
+            echo ""
```

Comment:
Indentation seems odd here?

Reply:
Fixed it ;) Can you check it looks good on your end?